### DISCUSSION TITLE
Will SDR++ need other drivers or apps to run Airspy HF+ Discovery on M1 Macs?

### DISCUSSION THREAD

**Discussion by NH7L** [AlexandreRouma/SDRPlusPlus#584] [General]
Very happy with SDR++ and how it runs an RTL-SDR v.3 on my M1 MacBook Air. When my Airspy HF+ Discovery arrives, will I need to install anything else to get it running on this machine?

**Comment by AlexandreRouma** [comment]
If you're using the .app from the nightly build (you should switch to it if you haven't yet), everything is included so it should work right out of the box.

**Reply by NH7L** [reply]
Thanks, Alexandre!

Ernie NH7L

**Reply by NH7L** [reply]
I must have done something wrong.  Downloaded  sdrpp_macos_amd64.pkg, which I assume is the nightly build,  and when I unpackaged it, never could find it anywhere on my M1 Mac. Installed a new copy of the version you gave me on Dec. 25 and got it running again.

**Reply by AlexandreRouma** [reply]
the pkg isn't the nightly build, it's the old release.

I forgot that I don't have the M1 build in the nightly build, only the intel .app is available.

The M1 .app I sent you should work until I can compile a newer build on m1

**Comment by NH7L** [comment]
Thanks for getting back to me, Alexandre. I can wait.  ;)

Ernie NH7L

**Comment by Jeffofmany** [comment]
I'm new to all this, including github. If I am in the wrong place or something let me know. I was wondering is sdr++ .app i have (V 1.0.6) will support the Airspy R2. Thank you.

**Reply by the-gobleo-tinkerer** [reply]
It should. If it doesn't work for you, open your own discussion thread to ask for help.

**Comment by NH7L** [comment]
Hi Alexandre —

After months away from my radios because of a long-distance move, I'm
interested in installing the newest working version of SDR++ on one of my
two M1 MacBook Air machines. Probably because of my own online
incompetence, I am not finding it on your Github pages. Can you tell me how
to get it? Or whether I even SHOULD get it, since I am still using the Feb.
6 2022 version I got from you? I'm guessing there may be updates/fixes/
improvements I can take advantage of.

For now, I plan to keep my other M1 AirBook running the Feb. 6 2022
version, which still runs perfectly. I used it until May, when I packed up
my belongings for the move from Washington state to Kansas. Just now got
settled into my newly rented flat. Once I have the newest version running
properly, I will copy it over to my second machine.

Still very grateful to you for all your hard work to give us Mac users an
excellent alternative to SDR#.

Thanks very much.

Ernest Murphy NH7L
***@***.*** or ***@***.***
Lawrence, Kansas USA

**Reply by AlexandreRouma** [reply]
Hi,

Unfortunately there isn't any new build for M1 because I still don't have access to one. They are still way too expensive to buy even used so I can't justify the expense. At some point I was thinking of buying the new M2 MacBook Air but my situation has changed and I decided to save up instead. GitHub said they would add M1 machines to the automated build systems which will allow me to build SDR++ for M1 but they still haven't delivered.

The only MacOS builds currently available are the [nightly builds](https://www.sdrpp.org/nightly)), but they are Intel only and have issues running on M1.

I have a friend who has an M1 machine so I may be able to get you a build soon enough but I won't be able to make the builds very often.

**Comment by NH7L** [comment]
Thanks for the quick reply, Alexandre.

I will continue using my copy of your old version of SDR++ for M1 Macs
then. Not at all a problem for me, since it works so well. I wish Macs were
more affordable. The cost is an issue for all who use them.

Thanks and 73!

Ernest Murphy NH7L
Lawrence, Kansas USA
